Makefile.am

```diff
@@ -791,7 +791,7 @@ lib_LTLIBRARIES += libtcmalloc_minimal_debug.la
 libtcmalloc_minimal_debug_la_SOURCES = src/debugallocation.cc \
                                        $(TCMALLOC_MINIMAL_INCLUDES)
 libtcmalloc_minimal_debug_la_CXXFLAGS = $(libtcmalloc_minimal_la_CXXFLAGS) \
-                                        -DTCMALLOC_FOR_DEBUGALLOCATION
```

Comment:
I don't think it is necessary. Sure, this define isn't used, but TCMALLOC_USING_DEBUGALLOCATION isn't supposed to be externally defined. It is used by debugallocator.cc before including (yes) tcmalloc.cc

Reply:
I noticed that TCMALLOC_USING_DEBUGALLOCATION is actually defined in debugallocation.cc.
I will revert this change.